certified-connectors/Data8/apiProperties.json

```diff
@@ -15,10 +15,39 @@
         }
       }
     },
+    "connectionParameterSets": {
```

Comment:
@mattybeard you are changing your connector form a single auth to a multiple auth connector type. I would suggest you wait maybe we can add this feature to your existing apiProperties I will get back to you very soon.

Reply:
Crossed wires working too quickly! I'll wait :)